backend/src/openarchiefbeheer/utils/utils_decorators.py

```diff
@@ -20,7 +20,7 @@ def decorator(func):
         sudo dockerd -H unix:///var/run/docker.sock -H tcp://127.0.0.1:2375
         ```
         """
-        if settings.ENVIRONMENT == "CI":
+        if settings.ENVIRONMENT == "development":
```

Comment:
What does this do?

Reply:
:scream: A mistake, I shouldn't have committed!